common/test/utils.py

```diff
@@ -0,0 +1,77 @@
+"""
+General testing utilies.
+"""
+
+from contextlib import contextmanager
+from django.dispatch import Signal
+from markupsafe import escape
+from mock import Mock, patch
+
+
+class XssTestMixin(object):
+    """
+    Mixin for testing XSS vulnerabilities.
+    """
+
+    def assert_xss(self, response, xss_content):
+        """Assert that `xss_content` is not present in the content of
+        `response`, and that its escaped version is present. Uses the
+        same `markupsafe.escape` function as Mako templates.
+
+        Args:
+          response (Response): The HTTP response
+          xss_content (str): The Javascript code to check for.
+
+        Returns:
+          None
+
+        """
+        self.assertContains(response, escape(xss_content))
+        self.assertNotContains(response, xss_content)
+
+
+def disable_signal(module, signal):
+    """Replace `signal` inside of `module` with a dummy signal. Can be
+    used as a method or class decorator, as well as a context manager."""
```

Comment:
Well, the intention is for the signal to have no side effects during the test. Since we're not making any assertions about the mock, like if it was called, I think it makes sense to call it "disable".